CONTRIBUTING.md

```diff
@@ -0,0 +1,336 @@
+- **Which external plugins or mods do you have installed?**
+- **Which Terra packs do you have installed?** You can get that list by running `/te packs`.
+- **Which Terra addons do you have installed?** You can get that list by running `/te addons`.
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion for Atom, including completely new features and minor improvements to
+existing functionality. Following these guidelines helps maintainers and the community understand your suggestion and find related
+suggestions.
+
+Before creating enhancement suggestions, please check [this list](#before-submitting-an-enhancement-suggestion) as you might find out that
+you don't need to create one. When you are creating an enhancement suggestion,
+please [include as many details as possible](#how-do-i-submit-a-good-enhancement-suggestion). Fill
+in [the template](https://github.com/atom/.github/blob/master/.github/ISSUE_TEMPLATE/feature_request.md), including the steps that you
+imagine you would take if the feature you're requesting existed.
+
+#### Before Submitting An Enhancement Suggestion
+
+- You must be on the **LATEST** version of Terra to make sure your feature hasn't been added yet.
+- Search for any [already existing issues](https://github.com/PolyhedralDev/Terra/issues?q=is%3Aissue+) (Including closed!) with your
+  problem. If you open a duplicate, it will be closed as such.
+- Verify that this is actually within the scope of Terra.
+- Be sure that this is not a feature request that should be made for a specific Terra *pack*, and instead applies to all of Terra.
+- Be sure that this is not something that should be implemented as a Terra addon, and instead applies to all of Terra.
+- Make sure that you attach a copy of the latest.log file, if there are any exceptions thrown in the console. Putting *just* the exception
+  **is not enough**. We need to be able to check that there wasn't anything else before that caused it.
+
+#### How Do I Submit A (Good) Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/). Create an issue on our main repository
+and provide the following information:
+
+- **Use a clear and descriptive title** for the issue to identify the suggestion.
+- **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+- **Provide specific examples to demonstrate the steps**.
+- **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
+- **Explain why this enhancement would be useful** to most Terra users and isn't something that can or should be implemented as an addon.
+
+### Pull Requests
+
+This section guides you through submitting a bug report for Terra. Following these guidelines helps maintainers and the community understand
+your report, and spend their time fixing the issue instead of understanding what you mean.
```

Comment:
This doesnt belong in the PR section

Reply:
Was just copying some stuff around because those 3 sections are more or less the same